src/schemas/components/SchemaFormPropertyAnyOf.vue

```diff
@@ -96,12 +96,14 @@
 
   const property = computed(() => {
     const selectedProperty = props.property.anyOf[selectedPropertyIndex.value]
+    // eslint-disable-next-line no-unused-vars
+    const { anyOf, ...property } = props.property
```

Comment:
unrelated change that strips the `anyOf` from the schema for an individual `anyOf` property. Wasn't hurting anything currently but would cause issues with validating a single property.